rclcpp/include/rclcpp/experimental/subscription_intra_process.hpp

```diff
@@ -64,11 +71,15 @@ class SubscriptionIntraProcess : public SubscriptionIntraProcessBase
     rclcpp::Context::SharedPtr context,
     const std::string & topic_name,
     rmw_qos_profile_t qos_profile,
-    rclcpp::IntraProcessBufferType buffer_type)
+    rclcpp::IntraProcessBufferType buffer_type,
+    std::shared_ptr<SerializationBase> serializer)
   : SubscriptionIntraProcessBase(topic_name, qos_profile),
-    any_callback_(callback)
+    any_callback_(callback), serializer_(serializer)
   {
-    if (!std::is_same<MessageT, CallbackMessageT>::value) {
+    if (!std::is_same<MessageT, CallbackMessageT>::value &&
+      !std::is_same<MessageT, rclcpp::experimental::SerializedContainer>::value &&
```

Comment:
How can that ever be true? That's to say whether `MessageT` is equal to `rcl_serialized_message_t`? That's never the case.

Reply:
for this implementation rcl_serialized_message_t is supported